auth-web/src/util/constants.ts

```diff
@@ -216,6 +216,7 @@ export enum Permission {
     DEACTIVATE_ACCOUNT='DEACTIVATE_ACCOUNT',
     VIEW_USER_LOGINSOURCE='VIEW_USER_LOGINSOURCE',
     EDIT_BUSINESS_INFO = 'EDIT_BUSINESS_INFO',
+    VIEW_DEVELOPER_ACCESS = 'VIEW_DEVELOPER_ACCESS',
```

Comment:
Need to create a migration file to add this to ADMIN role

Reply:
yeah..will do a PR now